### DISCUSSION TITLE
Chiusura Homix enel x

### DISCUSSION THREAD

**Discussion by volleyweb** [indomus/forum#3218] [GENERALE - Discussioni generiche]
Salve a tutti, Enel x chiuderà il supporto ad i server Homix e quindi tutti i dispositivi diventeranno dei fermacarte.
Per il modulo caldaia Homix pro HMXHBMEU02, qualcuno sà se può essere compatibile con altri sistemi od ha gia provato altri modi per sfruttarlo?
Pensavo ad home assistant.
Grazie.

**Comment by AntonioDiSipio** [comment]
Sarei curioso di saperlo anche io
